src/main/java/net/atos/zac/flowable/TakenService.java

```diff
@@ -210,20 +207,20 @@ private void createHistoricTaskLogEntry(
             historyService.createHistoricTaskLogEntryBuilder(task)
                     .type(type)
                     .data(FIELD_VISIBILITY_STRATEGY.toJson(new ValueChangeData(
-                            defaultString(oldValue, StringUtils.EMPTY),
```

Comment:
Is method `defaultString` used in many places? Then we should refactor (search & replace) as a separate PR...

Reply:
No other occurrences